front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/DisplayItinerary/Origin.tsx

```diff
@@ -104,6 +105,7 @@ function Origin(props: OriginProps) {
               className="btn btn-sm btn-only-icon btn-white"
               type="button"
               onClick={() => {
+                dispatch(clearVias());
```

Comment:
Why clear vias ? I don't think so, we should be able to change origin without clear all the settings of the path !